specs/template.md

```diff
@@ -117,10 +123,20 @@
 
 ---
 
-## Resources
+## Process
+
+### Checkpoints
+
+- [ ] **Checkpoint 1** - Schedule a spec review meeting to cover the Overview section before proceeding to work on the Design section. Be sure to include relevant stakeholders such as members of the tools team.
+
+- [ ] **Checkpoint 2** - Schedule a spec review meeting before beginning the implementation work. Be sure to invite relevant stakeholders such as design system owners.
+
+- [ ] **Checkpoint 3** - Submit the component PR and schedule a component code review meeting or present the component in one of our Engineering Open Mic sessions. Address feedback from the review, obtain final approvals through GitHub, and merge.
+
```

Comment:
What would you think about moving this towards the top? So at a glance, folks can see where this is "in process"

Reply:
I would agree with this, I think it could be at the very top, before the spec layout begins. It would also be informative at first glance, if we leave it in the spec, to inform on the status of the spec.

Reply:
Makes sense. I'll shuffle things a bit.